scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -110,11 +110,28 @@ package object hdfs {
 
     /** Save this SCollection as a text file on HDFS. */
     // TODO: numShards
-    def saveAsHdfsTextFile(path: String, username: String = null): Future[Tap[String]] = {
+    def saveAsHdfsTextFile(path: String,
+                           username: String = null,
+                           conf: Configuration = null): Future[Tap[String]] = {
+      val _conf = Option(conf).getOrElse {
```

Comment:
no need to go full monadic here.
`val _conf = if (conf != null) { conf } else {` is less cryptic?